pkgs/top-level/all-packages.nix

```
@@ -25717,6 +25717,13 @@ with pkgs;
     desktopName = "Firefox DevEdition";
   };
 
+  librewolf-unwrapped = firefoxPackages.librewolf;
+
+  librewolf = wrapFirefox librewolf-unwrapped {
+    inherit (librewolf-unwrapped) extraPrefsFiles extraPoliciesFiles;
+    libName = "librewolf";
+  };
```

```suggestion
  };
  librewolf-wayland = wrapFirefox librewolf-unwrapped { forceWayland = true; };
```

A wayland variant would be appreciated.

---

Added the wayland variant